man/systemd-system.conf.xml

```diff
@@ -162,6 +162,40 @@
         These settings have no effect if a hardware watchdog is not available.</para></listitem>
       </varlistentry>
 
+      <varlistentry>
+        <term><varname>RuntimeWatchdogPreSec=</varname></term>
+
+        <listitem><para>Configure the hardware watchdog device pre-timeout value.
+        Takes a timeout value in seconds (or in other time units similar to
+        <varname>RuntimeWatchdogSec=</varname>). A watchdog pre-timeout is a
+        notification generated by the watchdog before the watchdog reset might
+        occur in the event the watchdog has not been serviced. This notification
+        is handled by the kernel and can be configured to take an action (i.e.
+        generate a kernel panic) using the
+        <filename>/sys/class/watchdog/watchdog0/pretimeout_governor</filename>
```

Comment:
hmm, actually, thinking about this, I think we should just expose a knob for this here too. It's racy otherwise, as we probably should set the governor before turning on the logic.

i.e. add RuntimeWatchdogPreGovernor= that takes a string? and we just write the string into the file before fiddling with the pre timeout? should be a pretty tiny addition, and make this a lot simpler and easier to configure.